templates/ohdsi-webapi/atlas_database.bicep

```diff
@@ -0,0 +1,156 @@
+            }
+            { 
+              name: 'OHDSI_ADMIN_CONNECTION_STRING' 
+              secureValue: 'host=${postgresServer.properties.fullyQualifiedDomainName} port=5432 dbname=${databaseWebApiDatabaseName} user=${databaseWebapiAdminUsername} password=${databaseWebapiAdminPassword} sslmode=require'
+            }
+            {
+              name: 'DATABASE_NAME'
+              value: databaseWebApiDatabaseName
+            }
+            {
+              name: 'SCHEMA_NAME'
+              value: databaseSchemaName
+            }
+            {
+              name: 'OHDSI_ADMIN_PASSWORD'
+              secureValue: databaseWebapiAdminPassword
+            }
+            {
+              name: 'OHDSI_APP_PASSWORD'
+              secureValue: databaseWebapiAppPassword
+            }
+            {
+              name: 'OHDSI_APP_USERNAME'
+              value: databaseWebapiAppUsername
+            }
+            {
+              name: 'OHDSI_ADMIN_USERNAME'
+              value: databaseWebapiAdminUsername
+            }
+            {
+              name: 'OHDSI_ADMIN_ROLE'
+              value: databaseWebapiAdminRole
+
+            }
+            {
+              name: 'OHDSI_APP_ROLE'
+              value: databaseWebapiAppRole
+            }
+        ] 
+        scriptContent: ''' 
+            apk --update add postgresql-client
+            ADMIN_USER_PASSWORD="${OHDSI_ADMIN_PASSWORD}${OHDSI_ADMIN_USERNAME}"
+            APP_USER_PASSWORD="${OHDSI_APP_PASSWORD}${OHDSI_APP_USERNAME}"
+            ADMIN_MD5="'md5$(echo -n $ADMIN_USER_PASSWORD | md5sum | awk '{ print $1 }')'"
+            APP_MD5="'md5$(echo -n $APP_USER_PASSWORD | md5sum | awk '{ print $1 }')'"
+    
+            echo "CREATE ROLE ${OHDSI_ADMIN_ROLE} CREATEDB REPLICATION VALID UNTIL 'infinity'; COMMENT ON ROLE ${OHDSI_ADMIN_ROLE} IS 'Administration group for OHDSI applications'; CREATE ROLE ${OHDSI_APP_ROLE} VALID UNTIL 'infinity'; COMMENT ON ROLE ${OHDSI_APP_ROLE} IS 'Application groupfor OHDSI applications'; GRANT ${OHDSI_ADMIN_ROLE} TO ${OHDSI_ADMIN_USERNAME}; COMMENT ON ROLE ${OHDSI_ADMIN_ROLE} IS 'Admin user account for OHDSI applications'; CREATE ROLE ${OHDSI_ADMIN_USERNAME} LOGIN ENCRYPTED PASSWORD ${ADMIN_MD5} VALID UNTIL 'infinity'; GRANT ${OHDSI_ADMIN_ROLE} TO ${OHDSI_ADMIN_USERNAME}; COMMENT ON ROLE ${OHDSI_ADMIN_USERNAME} IS 'Admin user account for OHDSI applications'; CREATE ROLE ${OHDSI_APP_USERNAME} LOGIN ENCRYPTED PASSWORD ${APP_MD5} VALID UNTIL 'infinity'; GRANT ${OHDSI_APP_ROLE} TO ${OHDSI_APP_USERNAME}; COMMENT ON ROLE ${OHDSI_APP_USERNAME} IS 'Application user account for OHDSI applications'; GRANT ALL ON DATABASE ${DATABASE_NAME} TO GROUP ${OHDSI_ADMIN_ROLE}; GRANT CONNECT, TEMPORARY ON DATABASE ${DATABASE_NAME} TO GROUP ${OHDSI_APP_ROLE};" > ohdsi-part1.sql
```

Comment:
this can be done in one command, `psql "$MAIN_CONNECTION_STRING" -c "SQL statement."`

Reply:
Following our teams call, you can extract this bash script and load it using `loadTextContext('scripts/your_script.sh')`

You and can extract the SQL statements into `sql/your_sql_file.sql` path and use `supportingScriptUris` with a github branch parameter which defaults to v2 for readability. Replacing environment variables while loading the text from the sql files can be accomplished by `envsubst < your_sql_script.sql`.

Reply:
Done  :) 
Finally